enterprise/tools/check_metrics/check_metrics.go

```diff
@@ -201,3 +223,75 @@ func relativeRangeMetricHealthy(promAPI promapi.API, metricName string, metricVa
 	}
 	return false, status.InvalidArgumentErrorf("must specify a Within field for relative range thresholds")
 }
+
+type AttachmentBlock struct {
+	Blocks []SlackBlock `json:"blocks"`
+	Color  string       `json:"color"`
+}
+
+type SlackBlock struct {
+	Type string    `json:"type"`
+	Text TextBlock `json:"text"`
+}
+
+type TextBlock struct {
+	Type string `json:"type"`
+	Text string `json:"text"`
+}
+
+func sendMetricUnhealthySlack(metric string, isSecondaryMetric bool, maxSecondaryMetricFailureCount int) {
+	slackWebhookURL := os.Getenv("SLACK_WEBHOOK")
+	if slackWebhookURL == "" {
+		log.Warningf("No slack webhook set. Cannot send warning messages.")
```

Comment:
Errorf / Fatalf?

Reply:
Changed to Errorf. Didn't want to use Fatalf because this script causes a rollback in the deploy if it fails. If the env var accidentally gets unset and the slack can't be sent, it probably shouldn't trigger a rollback